tests/test_status.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import io
+import os
+import sys
+
+import generate_status_reference_data as gen
+import signac
+
+
+def test_print_status():
+
+    # Must import the data into the project.
+    with signac.TemporaryProject(name=gen.PROJECT_NAME) as p, signac.TemporaryProject(
+        name=gen.STATUS_OPTIONS_PROJECT_NAME
+    ) as status_pr:
+        gen.init(p)
+        fp = gen._TestProject.get_project(root=p.root_directory())
+        status_pr.import_from(origin=gen.ARCHIVE_PATH)
+        for job in status_pr:
+            kwargs = job.statepoint()
+            msg = f"---------- Status options {kwargs} for job {job}."
```

Comment:
Why is this this message added to both the generated and reference data?

Reply:
This is just a separator / header that makes it easier to tell how the outputs are generated.